packages/default-theme/src/components/SwProductDetails.vue

```diff
@@ -1,30 +1,31 @@
 <template>
-  <div class="sw-product-details">
+  <div class="sw-product-details" v-if="product">
     <div class="product-details__mobile-top">
       <SwProductHeading class="product-details__heading" :product="product" />
     </div>
     <SwPluginSlot name="product-page-description" :slot-context="product">
       <p class="product-details__description" v-html="description" />
     </SwPluginSlot>
-    <SfLoader :loading="isLoadingOptions">
-      <div v-if="product.options.length" class="product-details__section">
-        <div v-for="config in getOptionGroups" :key="config.id">
-          <SwProductSelect
-            v-if="getSelectedOptions[config.translated.name]"
-            :value="getSelectedOptions[config.translated.name]"
-            :options="getProductOptions({ product: config })"
-            :label="config.translated.name"
-            @change="
-              handleChange(
-                config.translated.name,
-                $event,
-                onOptionChanged($event)
-              )
-            "
-          />
-        </div>
+    <div
+      v-if="product.optionIds && product.optionIds.length"
+      class="product-details__section"
+    >
+      <div v-for="config in getOptionGroups" :key="config.id">
+        <SwProductSelect
+          v-if="getSelectedOptions[config.translated.name]"
+          :value="getSelectedOptions[config.translated.name]"
+          :options="getProductOptions({ product: config })"
```

Comment:
Is this helper class still required or can we remove it and use plain `config` instead? Because it will already contain all the options for that group. => simpler to read-.